pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/ActivityInstanceStatusDao.java

```diff
@@ -375,6 +372,37 @@ default void runPostActivityStatusChangeHooks(
         }
     }
 
+    private void processAnnouncementAction(Handle handle, long participantId, long studyId, EventConfigurationDto eventConfig) {
+        try {
+            UserAnnouncementDao announcementDao = getUserAnnouncementDao();
+            if (eventConfig.getAnnouncementCreateForProxies() == true) {
+                List<Governance> governances = handle.attach(UserGovernanceDao.class)
+                        .findActiveGovernancesByParticipantAndStudyIds(participantId, studyId)
+                        .collect(Collectors.toList());
+                if (!governances.isEmpty()) {
```

Comment:
I should know the answer to this, but I don't: is `governances` always going to be non-null?  If so, we can remove this check.

Reply:
JDBI will always give back a non-null `List`, but it might be empty so I'll keep this check.